iop/cdvd/cdfs/src/imports.lst

```diff
@@ -0,0 +1,64 @@
+cdvdman_IMPORTS_start
+I_sceCdInit
+I_sceCdGetError
+I_sceCdRead
+I_sceCdStop
+I_sceCdSync
+I_sceCdDiskReady
+I_sceCdGetDiskType
+I_sceCdTrayReq
+I_sceCdSearchFile
+cdvdman_IMPORTS_end
+
+intrman_IMPORTS_start
+I_CpuSuspendIntr
+I_CpuResumeIntr
+intrman_IMPORTS_end
+
+ioman_IMPORTS_start
+I_AddDrv
+I_DelDrv
+ioman_IMPORTS_end
+
+sysclib_IMPORTS_start
+I_tolower
+I_strcpy
+I_strncpy
+I_strncmp
+I_strtok
+I_strrchr
+I_strcat
+I_strlen
+I_memset
+I_memcpy
+I_memcmp
+sysclib_IMPORTS_end
+
+sifcmd_IMPORTS_start
+I_sceSifInitRpc
+I_sceSifSetRpcQueue
+I_sceSifRegisterRpc
+I_sceSifRpcLoop
+sifcmd_IMPORTS_end
+
+sifman_IMPORTS_start
+I_sceSifSetDma
+I_sceSifDmaStat
+sifman_IMPORTS_end
```

Comment:
Same here, all related to communication to the EE ?

Reply:
Removed!